lib/dfid-transition/transform/document.rb

```diff
@@ -83,6 +84,13 @@ def details
         #end
       end
 
+      def abstract
+        unescaped_abstract = DfidTransition::Transform::Html.unescape_three_times(
+          solution[:abstract].to_s)
+
+        Govuk::Presenters::Govspeak.present(unescaped_abstract)
```

Comment:
I don't think we want to do this here – we'd like the `abstract` property to reflect the (cleaned) string we get back from the triple store. Soon, the body won't just be the `abstract`, it'll be an amalgamation of things that *all* get fed to the presenter, so we'll need the call to `Govuk::Presenters::Govspeak#present` to remain in `#details`.